react-native-aztec/ios/RNTAztecView/RCTAztecView.swift

```diff
@@ -172,14 +174,35 @@ class RCTAztecView: Aztec.TextView {
         recognizer.isEnabled = false
     }
 
-    // MARK - View Height: Match to content height
+    // MARK: - View height and width: Match to the content
 
     override func layoutSubviews() {
         super.layoutSubviews()
+        measureWidth()
         fixLabelPositionForRTLLayout()
         updateContentSizeInRN()
     }
 
+    private func textSize(for width: CGFloat) -> CGSize {
+        let containerSize = CGSize(width: width, height: CGFloat.greatestFiniteMagnitude)
+        let container = NSTextContainer(size: containerSize)
+        let storage = NSTextStorage(attributedString: attributedText)
+        let layoutManager = NSLayoutManager()
+        layoutManager.addTextContainer(container)
+        storage.addLayoutManager(layoutManager)
+        let rawSize =  layoutManager.usedRect(for: container).size
+        return rawSize
+    }
+    
+    private func measureWidth() {
+        if(maxWidth > 0 && minWidth > 0) {
+            let newWidth = textSize(for: maxWidth).width
```

Comment:
I was wondering if `sizeThatFits:` can also work here which can save us from introducing the new method: `textSize`

Reply:
I remember that at some point of work I changed it to that function but i think that the `sizeThatFits:`  could work in the current implementation. Will try it and let you know :)

Reply:
Confirmed it works properly with `sizeThatFits`